docs/quickstarts/objective-c-quickstart.md

```diff
@@ -0,0 +1,294 @@
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+Let's update this so that the `Greeter` service has two methods. Edit `examples/proto/helloworld.proto` and update it with a new `SayHelloAgain` method, with the same request and response types:
+
+```
+// The greeting service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+  // Sends another greeting
+  rpc SayHelloAgain (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+```
+
+(Don't forget to save the file!)
+
+## Update the client and server
+
+We now have new gRPC service definition, but we still need to implement and call the new method in the human-written parts of our example application.
+
+### Update the server
+
+Recall that gRPC does not provide Objective-C version server API. We update the C++ sample server for our purpose. Open `examples/cpp/helloworld/greeter_server.cc`. Implement the new method like this:
+
+```
+class GreeterServiceImpl final : public Greeter::Service {
+  Status SayHello(ServerContext* context, const HelloRequest* request,
+                  HelloReply* reply) override {
+    std::string prefix("Hello ");
+    reply->set_message(prefix + request->name());
+    return Status::OK;
+  }
+  Status SayHelloAgain(ServerContext* context, const HelloRequest* request,
+                  HelloReply* reply) override {
+    std::string prefix("Hello again ");
+    reply->set_message(prefix + request->name());
+    return Status::OK;
+  }
+};
+```
+
+### Update the client
+
+Edit `examples/objective-c/helloworld/main.m` to call the new method like this:
+
+```
+int main(int argc, char * argv[]) {
+  @autoreleasepool {
+    [GRPCCall useInsecureConnectionsForHost:kHostAddress];
+    [GRPCCall setUserAgentPrefix:@"HelloWorld/1.0" forHost:kHostAddress];
+
+    HLWGreeter *client = [[HLWGreeter alloc] initWithHost:kHostAddress];
+
+    HLWHelloRequest *request = [HLWHelloRequest message];
+    request.name = @"Objective-C";
+
+    [client sayHelloWithRequest:request handler:^(HLWHelloReply *response, NSError *error) {
+      NSLog(@"%@", response.message);
+    }]; 
+    [client sayHelloAgainWithRequest:request handler:^(HLWHelloReply *response, NSError *error) {
+      NSLog(@"%@", response.message);
+    }]; 
+    
+    return UIApplicationMain(argc, argv, nil, NSStringFromClass([AppDelegate class]));
+  }
+}
+```
+
+Note that we made the second RPC call with the new gRPC service SayHelloAgain defined above.
```

Comment:
I think this sentence is a bit superfluous for a super-quick tutorial like this, we've already told them that they're calling the new method, they should be able to see it in the code.